### PR TITLE
Adjusted to FSiAdminBundle 1.1.*

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open

**Comment by rn0** [comment]
:+1:

File: Controller/PositionableController.php [hunk 1]
```diff
@@ -58,43 +65,48 @@ public function decreasePositionAction(CRUDElement $element, $id, Request $reque
...
         return new RedirectResponse($uri);
     }
 
     /**
-     * @param CRUDElement $element
+     * @param DataIndexerElement $element
      * @param $entity
      */
-    private function persistAndFlush(CRUDElement $element, $entity)
+    private function persistAndFlush(DataIndexerElement $element, $entity)
```
Comment (chives) [inline Controller/PositionableController.php]: this type hint could be changed to ``FSi\Bundle\AdminBundle\Doctrine\Admin\Element`` to ensure it has ``getObjectManager()`` method

Comment (szymach) [inline Controller/PositionableController.php]: True, but the element also [needs to have the getDataIndexer method.](https://github.com/szymach/admin-positionable-bundle/blob/master/Controller/PositionableController.php#L75). So the problem that neither of these interfaces have all the neccessary methods.

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Comment by chives** [comment]
:+1:

**Pull request merged**